### ISSUE TITLE
calculator freezes when running the program to install app.

### ISSUE THREAD

**Issue by d86592** [PeterTillema/Age-Of-CEmpires-I#27]
I compiled it correctly, and I added everything needed, but when my calculator runs the AOCE program, it freezes and I have to reset it to regain control of it.

**Comment by PeterTillema** [comment]
Yup, it broke after trying to replace parts in C. Should return back to assembly, which is at least more stable.

**Comment by d86592** [comment]
which commit should I go back to in order to make it work again?

**Comment by PeterTillema** [comment]
e9698a5c2ad590b8797d72be83b9d1c94eed6b33 is the latest working commit.

**Comment by d86592** [comment]
still seems to freeze. I downloaded that commit and compiled it with the same compiler, and deleted the old files off the calculator. Should I just wait for it to be fixed or am I doing something wrong?

**Comment by d86592** [comment]
still seems to freeze. I downloaded that commit and compiled it with the same compiler, and deleted the old files off the calculator. Should I just wait for it to be fixed or am I doing something wrong?

**Comment by Jaron-Wilson** [comment]
Yes It will glitch and it will not work on the calculators, If it does go all white, then at the back there is a button that says reset. Click that.
